presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -211,11 +220,18 @@ public static String textDistributedPlan(List<StageInfo> stages, Metadata metada
         // it's possible that some or all of them are missing or out of date.
         // For example, a LIMIT clause can cause a query to finish before stats
         // are collected from the leaf stages.
-        Map<PlanNodeId, Long> inputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> inputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> outputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> outputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> wallMillis = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeWallMillis = new HashMap<>();
+
+        Map<PlanNodeId, Map<String, Long>> operatorTotalDrivers = new HashMap<>();
```

Comment:
I have adjusted output as you suggested. Now it looks like:
```
             - InnerJoin[("orderkey" = "orderkey_0")][$hashvalue, $hashvalue_7] => [orderkey:bigint, $hashvalue:bigint, orderkey_0:bigint, $has
                     Cost: 39.15%, Output: 600572 rows (20.62MB)                                                                               
                     Left (probe) Input avg.: 150143.00 lines, Input std.dev.: 0.33%                                                           
                     Right (build) Input avg.: 9375.00 lines, Input std.dev.: 0.80%                                                            
                             Collisions avg.: 6244.50 (99.59% est.), Collisions std.dev.: 5.03%
```

However, I still wonder whether it makes sense to report `input avg`, `collisions avg` without the number of drivers that were running? I could just report `input std dev`, `collisions std dev`, but maybe it's better to display number of drivers with better name, say: `# of executors`?